### ISSUE TITLE
v2 require org-roam-graph.el error

### ISSUE THREAD

**Issue by Jousimies** [org-roam/org-roam#1549]
### Description
In my config, I required org-roam, but I cannot find org-roam-graph command. 
if i put (require 'org-roam-graph) into my config file, when startup ,there is a error Error running timer: (error "There is nothing to repeat"). 
But if I load the org-roam-graph.el after emacs startup, everything is right.
### Environment
All my config relate org-roam
`(require 'org-roam)
(setq org-roam-directory dn/galaxy-roam)
(setq org-roam-db-gc-threshold most-positive-fixnum)
(setq org-id-link-to-org-use-id t)
(org-roam-setup)
(setq org-roam-dailies-directory "journl/")
(setq org-roam-dailies-capture-templates
      '(("d" "default" entry
	 "* %?"
	 :if-new (file+head "journal/%<%Y-%m-%d>.org"
			    "#+title: %<%Y-%m-%d>\n"))))

(require 'org-roam-bibtex)
(add-hook 'org-roam-mode-hook #'org-roam-bibtex-mode)`

- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/commithashhere

**Comment by maikol-solis** [comment]
From this commit https://github.com/org-roam/org-roam/commit/6c9ba680a284261ea4779d2d6b76052e7a0216b9 `org-roam-graph` is working fine. 

Maybe try it with the latest commit on V2 and report a full debug trace to see what happened.

**Comment by aruscher** [comment]
@Jousimies I was able to reproduce your error:

### Steps to reproduce
Run this startup script `emacs -Q -l my-script.el`
``` emacs-lisp
(load-file "~/.emacs.d/straight/repos/straight.el/bootstrap.el")
(straight-use-package 'org-roam)

(require 'org-roam)
(setq org-roam-directory "~/ORG-MyLife/roam"
      org-roam-file-extensions '("org"))
(org-roam-setup)
(toggle-debug-on-error)
(require 'org-roam-graph)
````
### Backtrace

``` emacs-lisp
Debugger entered--Lisp error: (error "There is nothing to repeat")
  signal(error ("There is nothing to repeat"))
  error("There is nothing to repeat")
  repeat(string)
  (custom-declare-variable 'org-roam-graph-link-hidden-types (list 'funcall (list 'function #'(lambda nil '("file")))) "What sort of links to hide from the Org-roam graph..." :type (repeat 'string) :group 'org-roam)
  eval-buffer(#<buffer  *load*-124099> nil "/home/andreas/.emacs.d/straight/build/org-roam/org..." nil t)  ; Reading at buffer position 3254
  load-with-code-conversion("/home/andreas/.emacs.d/straight/build/org-roam/org..." "/home/andreas/.emacs.d/straight/build/org-roam/org..." nil t)
  require(org-roam-graph)
....
```

Seems to be related with #1552

**Comment by aruscher** [comment]
Should  be fixed by f989a6b

@Jousimies please check whether your error has gone away
@jethrokuan My test seems to work now. Please consider closing this issue.

**Comment by jethrokuan** [comment]
Closed with https://github.com/org-roam/org-roam/commit/f989a6bf8f478d1a7426f22ebdb11c18e2088412, feel free to reopen if it's still an issue.
